### PR TITLE
Add mobile ad sizes to desktop breakpoint

### PR DESCRIPTION
**Pull request by JonNorman**
## What does this change?
For an MVT, we are adding in banner sizes into the ad slots between gallery images (on desktop only).

When making the ad call, the sizes are taken from the maximum applicable breakpoint; it is not the aggregate applicable set of ad sizes i.e. mobile + desktop. As a result, we need to add the common sizes to the desktop breakpoint, otherwise (as is happening currently) only the banner ads are requested when on desktop and included in the test.)

## What is the value of this and can you measure success?
The test can go live and earn some more money from banner-competitive MPUs!

## Does this affect other platforms - Amp, Apps, etc?
No

## Tested in CODE?
No - tested locally.

@guardian/commercial-dev

### CODE REVIEW

**Comment by JonNorman** [comment]
[Fix gallery banner ads test](https://trello.com/c/QOooANrU/181-fix-gallery-banner-ads-test)

**Review by scottp719 — APPROVED** [review]
+1

File: applications/app/views/fragments/gallerySlot.scala.html [hunk 1]
```diff
@@ -5,22 +5,24 @@
 @import conf.switches.Switches.CommercialSwitch
 @import mvt.CommercialGalleryBannerAds
 
-@additionalSizes = @{
+@commonSizes = @{Seq("1,1", "2,2", "300,250", "fluid")}
+
+@additionalDesktopSizes = @{
```
Comment (kelvin-chappell) [inline applications/app/views/fragments/gallerySlot.scala.html]: This should probably either be called `desktopSizes` or not include `commonSizes`.

Comment (JonNorman) [inline applications/app/views/fragments/gallerySlot.scala.html]: Good point - I'll rename and merge.

**Pull request merged**